### PR TITLE
Development environment improvements

### PR DESCRIPTION
**Pull request by ddavidebor**
- Install python vscode automatically
- forward mariadb port so that debugging is easier
- Fix stale issues message
- add ssh-client to the dev image so that it's simpler to pass git credentials
- Add instructions to install an app
- add instructions to launch the Jupiter console in VSCode

### CODE REVIEW

File: development/README.md [hunk 1]
```diff
@@ -87,30 +95,52 @@ bench new-site sitename
...
 
 The command will ask the MariaDB root password. The default root password is `123`.
-This will create a new site and a `localhost` directory under `frappe-bench/sites`.
+This will create a new site and a `mysite.localhost` directory under `frappe-bench/sites`.
+You may need to configure your system /etc/hosts if you're on Linux, Mac, or its Windows equivalent.
+
+### Launching 
+
+To launch witho
+```shell
```
Comment (revant) [inline development/README.md:108]: "Witho"? Typo.

File: development/README.md [hunk 2]
```diff
@@ -87,30 +95,52 @@ bench new-site sitename
...
+bench new-site mysite.localhost
+```
+
```
Comment (revant) [inline development/README.md:111]: Bench start needed before accessing site in browser

File: .devcontainer/docker-compose.yml [hunk 3]
```diff
@@ -8,6 +8,8 @@ services:
     volumes:
       - ../installation/frappe-mariadb.cnf:/etc/mysql/conf.d/frappe.cnf
       - mariadb-vol:/var/lib/mysql
+    ports:
```
Comment (revant) [inline .devcontainer/docker-compose.yml:11]: I also have a native installed bench that I run along with dockerized env. If these ports are published I won't be able to start native Mariadb (linux)

Comment (ddavidebor) [reply .devcontainer/docker-compose.yml:11]: Good point, removed

**Comment by ddavidebor** [comment]
@revant thank you for your help

**Pull request merged**